### PR TITLE
Kc 6378 release notes for addons

### PR DESCRIPTION
**Pull request by vasanthsubram**
<https://koverse.atlassian.net/browse/KC-6378>

## why does this matter?
To easily see the changes to add-ons project

## how do you know this works?
verified by running docs locally

## how does this make you feel?
expression gif
Screen Shot 2020-10-12 at 4 20 01 PM

### CODE REVIEW

File: docs/source/releasenotes/add-ons/3.0.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Move koverse-addons to separate repo in Github
+
+
+Dependency upgrades
+-------------------
+
+3.0.4
+^^^^^
+
+changed koverse-sdk version to 3.4.1
```
Comment (etmarx) [inline docs/source/releasenotes/add-ons/3.0.rst:15]: Not sure that sdk version is necessary in the release notes. That is more of an implementation detail.

File: docs/source/releasenotes/add-ons/3.0.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.. _Version30ReleaseNotes:
```
Comment (etmarx) [inline docs/source/releasenotes/add-ons/3.0.rst:1]: Koverse release notes should specify the addons version starting with 3.6.0. Maybe add a 3.6.rst and start the 3.6.0 release notes with the addon version?

Comment (vasanthsubram) [reply docs/source/releasenotes/add-ons/3.0.rst:1]: fixed it

File: docs/source/releasenotes/add-ons/3.0.rst (hunk 1 above)
Comment (vasanthsubram) [reply docs/source/releasenotes/add-ons/3.0.rst]: removed the koverse-sdk version

**Review by etmarx — APPROVED** [review]

**Pull request merged**